### PR TITLE
fix some puppet style issues

### PR DESCRIPTION
**Pull request by tikitavi**

### CODE REVIEW

File: manifests/network/contrail/analytics.pp [hunk 1]
```diff
@@ -259,18 +259,18 @@
...
       'project_domain_name'     => $keystone_project_domain_name,
       'user_domain_name'        => $keystone_user_domain_name,
     }
     $auth_url_suffix = 'v3'
-    $vnc_authn_url = "/v3/auth/tokens"
+    $vnc_authn_url = '/v3/auth/tokens'
   }
   if $auth_protocol == 'https' {
-    $auth_url = "${auth_protocol}://${auth_host}:${auth_port_ssl}/${auth_url_suffix}"
+    $auth_url = "${auth_protocol}://${auth_host}:${auth_port}/${auth_url_suffix}"
```
Comment (alexey-mr) [inline manifests/network/contrail/analytics.pp]: Is it really correct change: auth_port_ssl => auth_port   ?

**Comment by tikitavi** [comment]
I’ve just reverted wrong change, occurred during cherry-pick from another branch.

File: manifests/network/contrail/analytics.pp (hunk 1 above)
Comment (tikitavi) [reply manifests/network/contrail/analytics.pp]: I’ve just reverted wrong change, occurred during cherry-pick from another branch.

Comment (tikitavi) [reply manifests/network/contrail/analytics.pp]: done

**Pull request merged**